### PR TITLE
Enable running the yml file from other repo

### PR DESCRIPTION
**Pull request by jenia81**
Use the context input parameter to enable running the action.yaml from MCC repo.

### CODE REVIEW

File: .github/workflows/PR-checker.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: production build
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```
Comment (JanneKiiskila) [inline .github/workflows/PR-checker.yml:7]: Wider scope? This runs ONLY for master and nothing else, it's not a PR check anymore.

Comment (jenia81) [reply .github/workflows/PR-checker.yml:7]: It runs on pull_request to master

**Review by JanneKiiskila — CHANGES_REQUESTED** [review]
Let's remove the commented out part, no point in keeping it there.

File: .github/actions/build-mbed-edge/action.yaml [hunk 2]
```diff
@@ -43,34 +27,13 @@ outputs:
...
-    # variables='';
-    # for i in $(env | grep '${{ inputs.env_pattern }}' | awk -F '=' '{print $1}'); do
-    #   variables="--env ${i} ${variables}";
-    # done;
     - name: Setup-Build-Files
       shell: bash
-      run: >
+      run: |
+        cd ${{ inputs.context }}
         cp -a .github/edge-config/${{ inputs.configfiles }}/* config
```
Comment (JanneKiiskila) [inline .github/actions/build-mbed-edge/action.yaml:34]: Improvement idea - let's move the certs to the scripts-internal to reduce visiblity to public.

**Review by JanneKiiskila — APPROVED** [review]

**Pull request merged**